lib/openQAcoretest.pm

```diff
@@ -28,6 +28,7 @@ sub test_flags { {fatal => 1} }
 
 sub upload_openqa_logs {
     get_log 'journalctl --pager-end --no-tail --no-pager -u apache2 -u nginx -u openqa-scheduler -u openqa-websockets -u openqa-webui -u openqa-gru -u openqa-worker@1' => 'openqa_services.log.txt';
+    get_log 'journalctl --pager-end --no-tail --no-pager' => 'journal.log.txt';
```

Comment:
Seems like overkill considering we already collect logs for services we care about - this is not "system" so much as everything including the "openQA services" 🤔

Reply:
I'd say either rename it to **Always upload the whole journal** or filter it to be a subset outside of what we already save

Reply:
you mean rename the git commit message?

Reply:
Yes. Just make it clear it's the whole journal. No strong opinion otherwise.

Reply:
ok, updated git commit message